feat: [issues/OCISDEV-305] Enable the mobile web view for OnlyOffice … (#11661)
owncloud/ocis: services/collaboration/README.md

  The product name of the connected WebOffice app, which can be one of the following:\
  `Collabora`, `OnlyOffice`, `Microsoft365` or `MicrosoftOfficeOnline`. This is used to internally control the behavior according to the different features of the used products.

* `COLLABORATION_APP_PRODUCT_EDITION`:\

>   The product edition of the connected WebOffice app define the capabilities specific to the product.
>   Supported OnlyOffice `ce` (community edition), `de` (developer edition), `ee` (enterprise edition) or `default:empty` equal to `ce`. This is used to internally control the behavior according to the different features of the used products.

Reviewer: ```suggestion
  The product edition of the WebOffice app defines the app's specific capabilities.
  The supported editions of OnlyOffice are: `ce` (community edition), `de` (developer edition), `ee` (enterprise edition) or `default:empty` which equals to `ce`. These are used to control internal behavior according to the features of the products used.
```

* `COLLABORATION_WOPI_SHORTTOKENS`:\
  Needs to be set if the office application like `Microsoft Office Online` complains about the URL is too long  (which contains the access token) and refuses to work. If enabled, a store must be configured.

* `COLLABORATION_WOPI_ENABLE_MOBILE`:\

>   The option enables the mobile view for office web app. This feature applies to OnlyOffice. The OnlyOffice `ce` only supports mobile view mode. The `ee` and `de` support mobile view in both view and edit modes. The product edition must be set accordingly via `COLLABORATION_APP_PRODUCT_EDITION`.

Reviewer: ```suggestion
  This option enables the mobile view for the Office Web App and only applies to OnlyOffice. While OnlyOffice `ce` supports mobile view mode only, the `ee` and `de` versions support mobile view in both view and edit modes. Set the product edition accordingly via `COLLABORATION_APP_PRODUCT_EDITION`.
```
